apps/src/sites/studio/pages/signup.js

```diff
@@ -230,6 +260,8 @@ window.SignupManager = function (options) {
     $("#user_terms_of_service_version").prop('checked', false);
 
     updateAutocompleteSchoolFields(schoolData);
+
+    sendSchoolInfoSplitTestEvent('signup-loaded');
```

Comment:
No `isTeacherSelected` needed here because this is within the `showTeacher` function